ui/src/components/Main.svelte

```diff
@@ -86,7 +93,7 @@
           <div class="w-[30%] flex items-end justify-center mb-[1vh]">
             {#if index === 0}
               <p class="px-[1.5vh] py-[0.4vh] rounded-full bg-primary text-[1.3vh]">
-                [E] Respond
+                [{RespondKeybind}] Respond
```

Comment:
In store.ts make a new store export const RESPOND_KEYBIND = writable<string>("");

Insted of [{RespondKeybind}] it should be [{$RESPOND_KEYBIND }] , note: you have to import the RESPOND_KEYBIND to the svelte file

Reply:
done